src/components/Footer/Footer.css

```diff
@@ -1,18 +1,19 @@
 :root {
```

Comment:
Move :root selector with all code inside of it to index.css file (the main file for very generic css code)